addon/ng2/commands/generate.ts

```diff
@@ -48,6 +91,32 @@ function mapBlueprintName(name) {
   return mappedName ? mappedName : name;
 }
 
+function importIntoAppModule(args, project, task) {
+  const pathObj = DynamicPathParser(project, args[1]);
+  const name = `${pathObj.name}.component`;
+  const camelizedName = `${name.slice(0, 1).toUpperCase()}${stringUtils.camelize(name).slice(1)}`;
+  const importPath = `./${pathObj.base}/${name}`;
+  const appModulePath = path.join(pathObj.appRoot, 'app.module.ts');
+  const appModuleContents = fs.readFileSync(appModulePath, 'utf8');
+
+  return new Promise((resolve, reject) => {
+    let parts = appModuleContents.split(EOL);
+    let imports = parts.filter(l => l.match(/import (.*)/));    
+    let declarationsStr = parts.filter(l => l.match(/declarations/))[0];
+    let idd = parts.indexOf(declarationsStr);
+    let currentDeclarations = declarationsStr.trim().match(/\[([^)]+)\]/)[1].split(',').map(d => d.trim());
```

Comment:
The blueprint has an `afterInstall` method where you can put the logic of adding the component to the app module.